msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Specify Microsoft 365 host runtime requirements in your app manifest
+description: Specify your app's runtime requirements in the app manifest to provide a deliberate experience on supported Microsoft 365 hosts. 
+ms.date: 6/21/2024
+ms.author: mosdevdocs
+author: erikadoyle
+ms.topic: conceptual
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+# Specify Microsoft 365 host runtime requirements in your app manifest (preview)
+
+TODO: Crosslinking to applicable reference and conceptual articles.
+
+> [!NOTE]
+>
+> The ability to specify Microsoft 365 host runtime requirements in the app manifest is currently in [public developer preview](../resources/schema/manifest-schema-dev-preview.md) and supported in Teams and Outlook mobile hosts.
+
+If you have a Teams app that contains a personal tab or message extension, you can easily [extend it to run across additional Microsoft 365 hosts](./overview.md), including Outlook and Microsoft 365 (Office) app across web, desktop, and mobile experiences. When you upgrade your Teams app to use app manifest version 1.13 or higher, your app is available in other Microsoft 365 application hosts by default. However, if your app also includes components that aren't yet supported across other Microsoft 365 hosts, your app might only partially load, resulting in unplanned and unsupported user experiences.
+
+To ensure intentional and deliberate customer experiences with your app, you can specify your app's runtime requirements in the app manifest to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts that you're not ready to support. Describing your app's runtime requirements in the app manifest also helps ensure your production-ready app experience will reach wider audiences in additional Microsoft 365 host applications in the future.
+
+Specifying your app's runtime requirements is useful in scenarios such as:
+
+
+- **One-way dependencies**: If your app includes a message extension component with an action command that requires bot-sent Adaptive Card functionality, you can ensure that particular command isn't available in message extension hosts that don't support bots, such as Outlook.
```

Comment:
Actually Outlook now even deprioritizes actionME support. So the botSentCard actionME scenario isn't an urgent one to support now. The botSentCard one's urgency is more on Copilot host support - actionME is now previewing in BizChat, but BizChat doesn't support bot so we want to omit the botSentCard actionME showing up in BizChat by not letting BizChat pick it as a skill.

That said, for narrative purpose for oneWayDepend, I suggest we use this example "have a tab that's a settings tab for app's ME which really contains the hero use case, and the tab shouldn't show on host not supporting ME, e.g. Harmony".  More details in RT spec.

Reply:
Sounds good--I swapped out the scenario for the suggested.